simcon_project/simcon_project/urls.py

```diff
@@ -27,6 +27,11 @@
     path('admin/', admin.site.urls),
     path('accounts/', include('django.contrib.auth.urls')),
     path('redirect-from-login/', redirect_from_login, name="redirect-from-login"),
+    path('password-reset/', password_reset, name="password_reset"),
+    path('password-reset/done/', views.PasswordResetDoneView.as_view(template_name='registration/password_reset_done.html'),name="password_reset_done"),
+    path('password-reset-confirm/<uidb64>/<token>/', views.PasswordResetConfirmView.as_view(template_name='registration/password_reset_confirm.html'),name="password_reset_confirm"),
+    path('password-reset-complete/', views.PasswordResetCompleteView.as_view(template_name='registration/password_reset_complete.html'),name="password_reset_confirm"),
```

Comment:
I'm gonna be a jerk here but for the url name, it should be hyphen-case instead of snake_case :)